web/firefoxcom.js

```diff
@@ -308,19 +308,53 @@ class FirefoxScripting {
 }
 
 class MLManager {
-  #enabled = new Map();
+  #enabled = null;
 
-  constructor({ enableAltText }) {
-    this.#enabled.set("altText", enableAltText);
+  eventBus = null;
+
+  constructor(options) {
+    this.enable({ ...options, listenToProgress: false });
   }
 
-  isEnabledFor(name) {
-    return this.#enabled.get(name);
+  async isEnabledFor(name) {
+    return !!(await this.#enabled.get(name));
```

Comment:
Given that `#enabled = null;` by default, can't this throw?
```suggestion
    return !!(await this.#enabled?.get(name));
```